framework/db/Connection.php

```diff
@@ -618,9 +625,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
```

Comment:
We are not using `\Yii` constuctions inside the code, it is better to import `Yii` class, so the line will be following:
```php
$command = Yii::createObject([
```